kernel/src/main/java/org/kframework/kompile/KompileOptions.java

```diff
@@ -49,7 +49,7 @@ GlobalOptions getGlobalOptions_UseOnlyInGuiceProvider() {
     public OutputDirectoryOptions outputDirectory = new OutputDirectoryOptions();
 
     // Common options
-    @Parameter(names="--backend", description="Choose a backend. <backend> is one of [llvm|haskell|kore|java]. Each creates the kompiled K definition.")
+    @Parameter(names="--backend", description="Choose a backend. <backend> is one of [llvm|haskell|kore]. Each creates the kompiled K definition.")
```

Comment:
The `kore` backend doesn't exist.

Reply:
It does, in fact. It emits a kore definition and then immediately stops. It's not really used in practice, but it does exist.